.circleci/config.yml

```diff
@@ -0,0 +1,8 @@
+version: 2.1
+
+jobs:
+  build:
+    docker:
+      - image: circleci/golang:1.12
+    steps:
+      - run: echo "Hello World"
```

Comment:
change to: "skipping build. PR https://github.com/mattermost/mattermost-server/pull/11978 in progress." or something like that